deepcell/applications/application_test.py

```diff
@@ -165,6 +167,14 @@ def _postprocess(Lx):
         y = app._postprocess([x])
         self.assertAllEqual(np.ones(x.shape), y)
 
+        # deep watershed
+        model_outputs = [np.zeros((3, 30, 30, 1)), np.zeros((3, 30, 30, 1)),
+                         np.zeros((3, 30, 30, 1))]
+        kwargs = {'postprocessing_fn': deep_watershed}
+        app = Application(model, **kwargs)
+        y = app._postprocess(model_outputs)
+        self.assertEqual(y.shape, model_outputs[0].shape)
+
```

Comment:
Is there a way to use a dummy post-processing step on this? I don't think we need to require functions outside of deepcell to test the code.

Reply:
For sure. If you look right above the new changes, you can see that Morgan already has such a dummy function. I thought adding this in might be good to catch any inadvertent changes that would break the post-processing, but if you think it's not necessary I can remove it

Reply:
I think the dummy meets the testing requirement, but you're welcome to include a test if it addresses another facet of post-processsing. My main concern is that it's testing `deep_watershed` more than it's testing the application itself.

Reply:
I guess what I was trying to address with this test is that our post-processing functions currently expect the model output to be a list of arrays, whereas the tests in the app for post-processing instead just pass a single array. But given that different post-processing functions could expect different outputs, it's probably not worth encoding that in the app itself?

Reply:
Maybe we can write a test for another dummy function that uses an array of length 2.

That way we can kinda test both lists and arrays generally for `Application`. That way subclasses can test their own post processing.